static/js/layout/editor.component.js

```diff
@@ -484,12 +490,7 @@ class EditorComponent {
       }
     });
 
-    this.editor.on('change', () => {
-      // GoldenLayout removes and re-inserts texts when dragging tabs, which
-      // in the end didn't change any of the contents, so we should ignore
-      // the change event in this case.
-      if (Terra.v.isDraggingTab) return;
-
+    this.editor.session.on('change', () => {
```

Comment:
This doesn't change anything. If I remove this `.session` and use `this.editor.on` in your branch and test it, then the on change is still not triggered, so you don't need this change. Perhaps this seems to be fixed with other changes we've made recently, so I suggest to just revert this change.

Reply:
I've changed it back because I couldn't reproduce.